locales/de.json

```diff
@@ -1,574 +1,911 @@
+      "description": "",
+      "help": "Elemente in der Fokusreihenfolge benötigen eine Rolle, die für interaktive Elemente geeignet ist."
+    },
+    "form-field-multiple-labels": {
+      "description": "Stellt sicher, dass ein form-Feld nur ein label-Element besitzt.",
+      "help": "form-Felder sollten nur ein label-Element besitzen."
+    },
+    "frame-tested": {
+      "description": "",
+      "help": "Frames müssen mit axe-core getestet werden."
+    },
+    "frame-title-unique": {
+      "description": "",
+      "help": "Frames müssen ein einzigartiges title-Attribut besitzen."
+    },
+    "frame-title": {
+      "description": "",
+      "help": "Frames müssen ein title-Attribut besitzen."
+    },
+    "heading-order": {
+      "description": "",
+      "help": "Überschriftenebenen sollten nur jeweils um eins steigen."
+    },
+    "hidden-content": {
+      "description": "",
+      "help": "Versteckter Inhalt auf der Seite konnte nicht analysiert werden."
+    },
+    "html-has-lang": {
+      "description": "",
+      "help": "Das <html>-Element muss ein lang-Attribut besitzen."
+    },
+    "html-lang-valid": {
+      "description": "",
+      "help": "Das <html>-Element muss einen gültigen Attributwert für das lang-Attribut besitzen."
+    },
+    "html-xml-lang-mismatch": {
+      "description": "Stellt sicher, dass HTML Elemente mit validen lang und xml:lang Attributen diesselbe Angabe über die Sprache machen.",
+      "help": "HTML Elemente mit lang und xml:lang Attributen müssen diesselbe Sprache ausweisen."
+    },
+    "identical-links-same-purpose": {
+      "description": "Stellt sicher, dass links mit dem selben Namen (accessible-name) denselben Zweck folgen.",
+      "help": "Links mit dem selben Namen verfolgen denselben Zweck."
+    },
+    "image-alt": {
+      "description": "",
+      "help": "Abbildungen müssen einen Alternativtext besitzen."
+    },
+    "image-redundant-alt": {
+      "description": "",
+      "help": "Der Text von Schaltflächen oder Links sollte in der Abbildungsalternative nicht wiederholt werden."
+    },
+    "input-button-name": {
+      "description": "Stellt sicher, dass Eingabeschaltflächen sichtbaren Text beinhalten.",
+      "help": "Eingabeschaltflächen müssen sichtbaren Text beinhalten."
+    },
+    "input-image-alt": {
+      "description": "",
+      "help": "Bild-Schaltflächen müssen einen Alternativtext besitzen."
+    },
+    "label-content-name-mismatch": {
+      "description": "Stellt sicher, dass Elemente die durch ihren Inhalt beschrieben sind auch ihren sichtbaren Text als Teil des zugänglichen Namens haben.",
+      "help": "Elemente müssen ihren sichtbaren Text auch als Teil des zugänglichen Namens haben."
+    },
+    "label-title-only": {
+      "description": "",
+      "help": "Formularelemente sollten eine sichtbare Beschriftung haben."
+    },
+    "label": {
+      "description": "",
+      "help": "Formularelemente müssen eine Beschriftung haben."
```

Comment:
This is a good translation but a more accurate meaning would be `Formularelemente müssen einen barrierfreien-Namen besitzen`

Reply:
Thoughts on this last one?

Reply:
I disagree, because the label of a element is clearly "Beschriftung eines Elements".